samples/net/wpan_serial/README.rst

```diff
@@ -0,0 +1,54 @@
+.. _wpanusb_serial-sample:
+
+802.15.4 "serial-radio" sample
+##############################
+
+Overview
+********
+
+The wpan_serial sample shows how to use hardware with 802.15.4 radio and USB
+controller as a "serial-radio" device for Contiki - based border routers.
+
+Requirements
+************
+
+The sample assumes that 802.15.4 radio and USB controller are supported on
+a board.
+
+Building and Running
+********************
+
+#. Build and flash the sample Zephyr application to a board
+   with a 802.15.4 radio and USB controller
+   (such as quark_se_c1000_devboard)
```

Comment:
Is it possible to use a board that's still available in the market for this example?  I don't think you can't buy these Quark-based boards anymore.

Reply:
You can buy another board with USB controller and 802.15.4 chip supported by Zephyr.